airflow/api_connexion/endpoints/role_and_permission_endpoint.py

```diff
@@ -40,14 +40,14 @@
 
 if TYPE_CHECKING:
     from airflow.api_connexion.types import APIResponse, UpdateMask
-    from airflow.www.security import AirflowSecurityManager
+    from airflow.www.security_manager import AirflowSecurityManagerV2
```

Comment:
Since on the long term we will deprecate `AirflowSecurityManager` in favor of `AirflowSecurityManagerV2` I am wondering if `AirflowSecurityManagerV2` is a good name since this name will stay. Should we keep the name `AirflowSecurityManager`? It can confusing but we can assume this is only temporary? Or should we come up with another name? `SimplifiedAirflowSecurityManager`? I dont like it either xD.

Reply:
temporary things still stay around for a pretty long time to give users time to migrate...
Idk, I'm open to suggestions, but we can always rename it once we remove the old one.